src/main/java/frc/robot/Path.java

```diff
@@ -10,10 +10,13 @@
     private ArrayList rightString = new ArrayList<>();
     private ArrayList leftString = new ArrayList<>();
```

Comment:
Since you are storing Strings in these ArrayLists, the instantiation should be "ArrayList<String> leftString = new ArrayList<String>();"